.h5pignore

```diff
@@ -12,3 +14,6 @@ storybook-static
 docs
 babel.config.js
 tsconfig.json
+LICENSE
+jest.config.ts
+typings.d.s
```

Comment:
```suggestion
typings.d.ts
```

Reply:
⚠️ Add `.github`, `build_info` and `h5p.d.ts` as well.

Reply:
Added.